project/src/main/webapp/index.html

```diff
@@ -25,8 +25,14 @@
         <br>
         <input type="text" id="address" name="address" placeholder="...your street address..."
                 required/>
+        <input type="hidden" name="listAll" value="false"/>
         <input type="submit" id="submit-address"/>
       </form>
+      <form id="list-all-input" action="/directory.html">
+        <input type="hidden" name="address" value="default"/>
```

Comment:
Maybe make the "value" attribute more descriptive, such as "street address"/"residence address"